local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -3,15 +3,20 @@
 LocalPlannerNode::LocalPlannerNode() {
   nh_ = ros::NodeHandle("~");
   readParams();
+  newest_cloud_msgs_.resize(3);
 
   // Set up Dynamic Reconfigure Server
   dynamic_reconfigure::Server<avoidance::LocalPlannerNodeConfig>::CallbackType
       f;
   f = boost::bind(&LocalPlannerNode::dynamicReconfigureCallback, this, _1, _2);
   server_.setCallback(f);
 
-  pointcloud_sub_ = nh_.subscribe<const sensor_msgs::PointCloud2&>(
-      depth_points_topic_, 1, &LocalPlannerNode::pointCloudCallback, this);
+  pointcloud_front_sub_ = nh_.subscribe<sensor_msgs::PointCloud2>(
```

Comment:
I'd like this to be more generic:
- Add a struct with all data for a single camera (point cloud subscription, newest_cloud_msg, ...)
- Then depending on the configured cameras (check `nh_.param<std::string>("depth_points_topic_front", depth_points_topic_front_,  "/camera_front/depth/points");`, initialize a vector of that struct and do the subscriptions.
- Then pass the number of cameras to all classes that need to know it and initialize the vectors for example with `(num_cameras, false)`, instead of the hardcoded size `{false, false, false}`.